apps/src/applab/gridUtils.js

```diff
@@ -25,8 +25,8 @@ export function scaledDropPoint(draggedElement) {
   const xScale = boundingRect.width / div.offsetWidth;
   const yScale = boundingRect.height / div.offsetHeight;
 
-  let left = (draggedOffset.left - boundingRect.left) / xScale;
-  let top = (draggedOffset.top - boundingRect.top) / yScale;
+  let left = (draggedOffset.left - $(div).offset().left) / xScale;
+  let top = (draggedOffset.top - $(div).offset().top) / yScale;
 
```

Comment:
I made this change here because using `boundingRect.left` and `boundingRect.top` returned strange values in tests (and only in the tests; can't repro when running in browser) when an element had just been dragged in to the bottom-right-ish area of the app space using the `dragToVisualization` helper. 

From debugging the test, I found that `boundingRect.top` returned `top` position values for `#designModeViz` that are shifted by 20px/40px depending on the y value passed into `dragToVisualization`.

It's the **weirdest** thing, and I still can't figure out why. 

Reading the code, it looks like the method is trying to find the position of `#designModediv` relative to the screen. `.offset().left` and `.offset().top` does the same thing, and didn't do funny things in the tests, so I replaced it here.